tests/scripts/analyze_outcomes.py

```diff
@@ -212,6 +212,332 @@ def do_analyze_driver_vs_reference(outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_cipher_aead': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_cipher_aead',
+            'component_driver': 'test_psa_crypto_config_accel_cipher_aead',
+            # Ignore suites that are being accelerated
+            'ignored_suites': [
+                'aes.cbc',
+                'aes.cfb',
+                'aes.ecb',
+                'aes.ofb',
+                'aes.rest',
+                'aes.xts',
+                'aria',
+                'camellia',
+                'ccm',
+                'chacha20',
+                'chachapoly',
+                'cipher.aes',
+                'cipher.aria',
+                'cipher.camellia',
+                'cipher.ccm',
+                'cipher.chacha20',
+                'cipher.chachapoly',
+                'cipher.des',
+                'cipher.gcm',
+                'cipher.nist_kw',
+                'cipher.padding',
+                'des',
+                'gcm.aes128_de',
+                'gcm.aes128_en',
+                'gcm.aes192_de',
+                'gcm.aes192_en',
+                'gcm.aes256_de',
+                'gcm.aes256_en',
+                'gcm.camellia',
+                'gcm.misc',
+            ],
+            'ignored_tests': {
+                # Following tests depends on AES_C/DES_C
+                'test_suite_pem': [
+                    'PEM read (AES-128-CBC + invalid iv)'
+                    'PEM read (DES-CBC + invalid iv)',
+                    'PEM read (DES-EDE3-CBC + invalid iv)',
+                    'PEM read (malformed PEM AES-128-CBC)',
+                    'PEM read (malformed PEM DES-CBC)',
+                    'PEM read (malformed PEM DES-EDE3-CBC)',
+                    'PEM read (unknown encryption algorithm)',
+                    'PEM read (AES-128-CBC + invalid iv)',
+                    'PEM read (DES-CBC + invalid iv)',
+                ],
+                # Following tests depends on AES_C/DES_C
+                'test_suite_error': [
+                    'Low and high error',
+                    'Single low error'
+                ],
+                # Following tests depends on AES_C/DES_C/GCM_C/CTR
```

Comment:
On second thought, this part (`test_suite_psa_crypto`) is not really satisfying: do these tests really need the built-in implementations? If so, the reason should be explained here. If not, the dependency declarations should be fixed. (Could be a mix of both depending on why test, or course.)

Reply:
None of the test cases here under `test_suite_psa_crypto` should depend on the built-in implementation.

I have not checked whether it makes more sense to add an exception now and fix them later, or fix them now.